src/main/java/salesforce/ui/pages/contacts/ContactsFormPage.java

```diff
@@ -0,0 +1,242 @@
+/**
+ * Copyright (c) 2021 Fundacion Jala.
+ * This software is the confidential and proprietary information of Fundacion Jala
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * license agreement you entered into with Fundacion Jala
+ */
+
+package salesforce.ui.pages.contacts;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import salesforce.ui.pages.BasePage;
+import java.util.HashMap;
+
+public class ContactsFormPage extends BasePage {
+
+    @FindBy(css = "input[placeholder='Search Accounts...']")
+    private WebElement accountSearchBox;
+
+    @FindBy(css = "input[placeholder='Search Contacts...']")
+    private WebElement contactsSearchBox;
+
+    @FindBy(xpath = "(//textarea[@name='street'])[1]")
+    private WebElement mailingStreetTxtBox;
+
+    @FindBy(xpath = "(//input[@name='postalCode'])[1]")
+    private WebElement mailingPostalCodeTxtBox;
+
+    @FindBy(xpath = "(//input[@name='city'])[1]")
+    private WebElement mailingCityTxtBox;
+
+    @FindBy(xpath = "(//input[@name='province'])[1]")
+    private WebElement mailingProvinceTxtBox;
+
+    @FindBy(xpath = "(//input[@name='country'])[1]")
+    private WebElement mailingCountryTxtBox;
+
+    @FindBy(xpath = "(//textarea[@name='street'])[2]")
+    private WebElement otherStreetTxtBox;
+
+    @FindBy(xpath = "(//input[@name='postalCode'])[2]")
+    private WebElement otherPostalCodeTxtBox;
+
+    @FindBy(xpath = "(//input[@name='city'])[2]")
+    private WebElement otherCityTxtBox;
```

Comment:
Some xpath are repeated, maybe you could refactor them. Also you should avoid using indexes on the locators.